docs/src/components/Palette.tsx

```diff
@@ -0,0 +1,282 @@
+      onBlur={() => setIsHovered(false)}
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}
+      order={order}
+      full={full}
+      left={left}
+      right={right}
+    >
+      <ColorNameHolder isMain={isNormal(color.name)}>{color.name}</ColorNameHolder>
+      <ColorHexHolder>{color.value}</ColorHexHolder>
+      <ColorOtherTextHolder>{convertHexToRgba(color.value, 100)}</ColorOtherTextHolder>
+      <ColorOtherTextHolder>{color.tokenName}</ColorOtherTextHolder>
+      {isHovered && (
+        <CopyWrapper colorValue={color.value}>
+          <Stack direction="column" spacing="XXSmall" align="end">
+            <CopyButton textToCopy={color.value} colorValue={color.value} buttonText="Hex" />
+            <CopyButton
+              textToCopy={convertHexToRgba(color.value, 100)}
+              colorValue={color.value}
+              buttonText="RGB"
+            />
+            <CopyButton textToCopy={color.tokenName} colorValue={color.value} buttonText="Token" />
+          </Stack>
+        </CopyWrapper>
+      )}
+    </ColorContainerWrapper>
+  );
+};
+
+interface PaletteProps {
+  colors: tokenString[];
+  allowAdditional: boolean;
+}
+
+const normalizeName = (tokenName: string) =>
+  tokenName
+    .replace(/^palette/, "")
+    .replace(/^color/, "")
+    .replace(/([a-z])([A-Z])/, "$1 $2")
+    .replace("Hover", ":hover")
+    .replace("Active", ":active");
+
+const Palette = ({ colors, allowAdditional }: PaletteProps) => {
+  const [showAdditional, setShowAdditional] = React.useState(false);
+  const { isTablet } = useMediaQuery();
+  const switchShowAdditional = () => {
+    setShowAdditional(!showAdditional);
+  };
+
+  const colorsWithToken = colors.map(color => ({
+    name: normalizeName(color),
+    tokenName: color,
+    value: defaultTokens[color],
+    hover: {
+      name: normalizeName(`${color}Hover`),
+      tokenName: `${color}Hover`,
+      value: defaultTokens[`${color}Hover`],
+    },
+    active: {
+      name: normalizeName(`${color}Active`),
+      tokenName: `${color}Active`,
+      value: defaultTokens[`${color}Active`],
+    },
+  }));
+  return (
+    <Stack>
+      {allowAdditional && (
+        <Switch onChange={switchShowAdditional} checked={showAdditional} reverseLabel>
+          Show additional shades
+        </Switch>
+      )}
+      <div>
+        {colorsWithToken.map((color, index, allColors) => {
+          const determineOrder = indexNumber => {
+            if (allColors.length === 1) return "only";
+            if (indexNumber === 0) return "first";
+            if (indexNumber === allColors.length - 1) return "last";
+            return "middle";
+          };
+          const hasAdditional = obj => {
+            if (obj.hover.value || obj.active.value) return true;
+            return false;
+          };
+          return (
+            <Grid columns={showAdditional && color.hover.value && isTablet ? "34% 33% 33%" : "1fr"}>
```

Comment:
is there some reason, why first column of grid should be `34%`? I think it will enough to have just `1fr 1fr 1fr`